src/pineko/check.py

```diff
@@ -172,34 +185,50 @@ def contains_ren(grid, max_as, max_al):
         is ren scale-variation available for as
     bool
         is ren scale-variation available for al
+    bool
+        is the central as order reached by the grid
+    bool
+        is the central al order reached by the grid
     """
     order_array = np.array([order.as_tuple() for order in grid.orders()])
     order_mask = pineappl.grid.Order.create_mask(grid.orders(), max_as, max_al)
     order_list = order_array[order_mask]
     as_orders = []
     al_orders = []
     for order in order_list:
-        as_orders.append(order[0])
-        al_orders.append(order[1])
-    min_as = min(as_orders)
-    min_al = min(al_orders)
+        if order[1] == 0:
+            as_orders.append(order)
+        else:
+            al_orders.append(order)
+    min_as = min([ord[0] for ord in as_orders]) if len(as_orders) != 0 else 0
+    min_al = min([ord[0] for ord in al_orders]) if len(al_orders) != 0 else 0
     order_as_is_present = False
     order_al_is_present = False
     sv_as_present = False
     sv_al_present = False
-    # ren sv starts one order after the first order with as if the first order in as is not as^0
-    add_order = 1 if min_as != 0 else 2
-    for order in order_list:
-        if order[0] == min_as + add_order:
+    # ren sv starts one order after the first order with as (al) if the first order in aa (al) is not as^0 (al^0)
+    add_order_as = 1 + (max_as - 2)
```

Comment:
Why not just `-1`?

Reply:
Of course is the same. However, in this case I wanted to make clear that `+1` means what the comment says above, i.e. that ren sv starts one order later than the first non zero order. Then, because of our convention on `max_as`, I need to do `max_as -2`. If I just write `max_as - 1` it would not be clear (because it won't be consistent with the comment).